projects/plugins/jetpack/extensions/blocks/subscriber-login/block.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 1,
+	"name": "jetpack/subscriber-login",
+	"title": "Subscriber Login",
+	"description": "Subscriber Login",
+	"keywords": [],
+	"version": "0.0.1",
+	"textdomain": "jetpack",
+	"category": "grow",
+	"icon": "<svg viewBox='0 0 32 32' width='32' height='32' xmlns='http://www.w3.org/2000/svg'><path d='M9 15h2V9H9v6zm1-10c-.5 0-1 .5-1 1s.5 1 1 1 1-.5 1-1-.5-1-1-1zm0-4c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9zm0 16c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z'/></svg>",
+	"supports": {
+		"align": false,
+		"alignWide": true,
```

Comment:
I think left/right/center align would be good for this block (`align: true`), but `alignWide` I doubt is helpful for the block.

Reply:
Actually curious what `supports` core login block has, and imitate.

Reply:
Thanks! Added some styling support here https://github.com/Automattic/jetpack/pull/35085